docs/api/system-preferences.md

```diff
@@ -327,6 +327,12 @@ You can use the `setAppLevelAppearance` API to set this value.
 Sets the appearance setting for your application, this should override the
 system default and override the value of `getEffectiveAppearance`.
 
+### `systemPreferences.isTrustedAccessibilityClient(prompt)` _macOS_
+
+* `prompt` Boolean - whether or not the user will be informed via prompt if the current process is untrusted.
+
+Returns `Boolean` - `true` if the current process is a trusted accessibility client and `false` if it is not.
```

Comment:
Does this return before or after the user is prompted? E.g. if the user opts to accept the prompt and add the app, will this return `true`?

Reply:
the prompt can't be accepted per se, it just links to the system prefpane; so no :)